### PR TITLE
Added Ukrainian Translation

### PR DESCRIPTION
**Pull request by taras-taras**
Just started using git, please tell me if I did something wrong.

### CODE REVIEW

**Comment by agateau** [comment]
Hi, thanks for this translation! It looks good, only missing a few strings.

screenshot-2022-06-02-235555 218

screenshot-2022-06-02-235609 754

screenshot-2022-06-02-235639 149

I added a commit (b9c6fd4a80ede0021e5af4bd6e54ac386d2f0fae) to add the missing entry to languages.xml so that the game can load your translation, but the "Ukrainian" word itself should be translated in Ukrainian (it appears in the language list you see on the third screenshot). Would be great if you could update the languages.xml file. You can also tell me how to say "Ukrainian" in Ukrainian instead, and I'll do the change myself.

File: android/assets/po/uk.po [hunk 1]
```diff
@@ -0,0 +1,843 @@
...
+
+#: core/src/com/agateau/pixelwheels/screens/UnlockedRewardScreen.java:120
+#, fuzzy
+msgid   "New championship unlocked!"
+msgstr  "Чемпіонат\n"
+        "Завершено!"
+
+#: core/src/com/agateau/pixelwheels/utils/StringUtils.java:46
+msgid   "1st"
+msgstr  "1-й "
```
Comment (Nickoriginal) [inline android/assets/po/uk.po:815]:
```suggestion
msgstr  "1-й"
```

File: android/assets/po/uk.po [hunk 2]
```diff
@@ -0,0 +1,843 @@
...
+
+#: core/src/com/agateau/pixelwheels/utils/StringUtils.java:48
+msgid   "2nd"
+msgstr  "2-й"
+
+#: core/src/com/agateau/pixelwheels/utils/StringUtils.java:50
+msgid   "3rd"
+msgstr  "3-й "
```
Comment (Nickoriginal) [inline android/assets/po/uk.po:823]:
```suggestion
msgstr  "3-й"
```

File: android/assets/po/uk.po [hunk 3]
```diff
@@ -0,0 +1,843 @@
...
+
+#: core/src/com/agateau/pixelwheels/utils/StringUtils.java:52
+msgid   "4th"
+msgstr  "4-й "
```
Comment (Nickoriginal) [inline android/assets/po/uk.po:827]:
```suggestion
msgstr  "4-й"
```

**Review by Nickoriginal — CHANGES_REQUESTED** [review]
Fix some mistypes:

File: android/assets/po/uk.po [hunk 4]
```diff
@@ -0,0 +1,843 @@
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+#
+# taras-taras <taraspavliuk@protonmail.com>, 2022.
+msgid   ""
+msgstr  "Project-Id-Version: \n"
+        "Report-Msgid-Bugs-To: \n"
+        "POT-Creation-Date: 2022-01-17 23:21+0100\n"
```
Comment (Nickoriginal) [inline android/assets/po/uk.po:8]:
```suggestion
```

**Review by Nickoriginal — CHANGES_REQUESTED** [review]
Remove POT-Creation-Date (see https://github.com/agateau/pixelwheels/pull/233#issue-1236085874)

File: android/assets/po/uk.po [hunk 5]
```diff
@@ -0,0 +1,843 @@
...
+msgid   "Antonin Gâteau"
+msgstr  "Antonin Gâteau"
+
+#: android/assets/screens/credits.gdxui:19
+msgid   "Music"
+msgstr  "Музика"
+
+#: android/assets/screens/credits.gdxui:21
+msgid   "Menu:"
+msgstr  "Меню: "
```
Comment (Nickoriginal) [inline android/assets/po/uk.po:105]:
```suggestion
msgstr  "Меню:"
```

**Comment by Nickoriginal** [comment]
Also @taras-taras, you can see https://github.com/agateau/pixelwheels/pull/228#issuecomment-1122007096 and https://github.com/agateau/pixelwheels/pull/218#issuecomment-1102194092 to learn the sense of the tricky names.

**Comment by agateau** [comment]
> Looks like it is stuck... Closing or trying to merge?

I already tried to reach out to Taras but haven't heard back yet. I am going to leave this PR open for now. It's too late for 0.23.0 anyway, but a few translations are outdated, so I may do a 0.23.1 in a month if I can get a few of them updated. If I heard from Taras in the mean time, this can be merged.

**Comment by agateau** [comment]
Sadly, this PR looks it's not moving anymore :/. Closing it for now.

**Pull request closed without merging**